### ISSUE TITLE
Syntax highlighting for C++ classes fails if base classes are declared on next line

### ISSUE THREAD

**Issue by aeschli** [atom/language-c#274]
_From @elpixo on April 26, 2018 13:3_

Syntax highlighting for C++ classes fails if base classes are declared on next line.

```
// This is highlighted correctly
class SomeClass : public BaseClass 
{
};

// This has no syntax highlight for public BaseClass
class SomeClass 
   : public BaseClass
{
};
```

_Copied from original issue: Microsoft/vscode#48756_

**Comment by Arcanemagus** [comment]
This is potentially fixed by #252, as it is marked as fixing several issues related to this.

**Comment by maxbrunsfeld** [comment]
This is fixed if you use the new [Tree-sitter](http://tree-sitter.github.io/tree-sitter) parsing system, which is enabled by default in Atom 1.32 Beta. If you're not the beta channel, you can enable it by checking the `Use Tree-sitter Parsers` box in the settings view, or wait for Atom 1.32 Stable to be released. Thanks!
